src/js/components/pages.js

```diff
@@ -387,18 +387,17 @@ export class NoticeIndex {
 
 export class NoticePage {
   constructor(content) {
+    this.$element = $(".styles .grid--fullwidth").clone();
```

Comment:
Modal pages, which this one is, empties out the container(`this.page.empty();`) before appending and so in doing so, it removes `.grid--default`. This seems to work as well though.